### ISSUE TITLE
Malformed JSON when target host in local subnet is down

### ISSUE THREAD

**Issue by jakkanen** [traviscross/mtr#190]
Given host 192.168.1.4 is in the same subnet but down, the following will result in malformed JSON:
`mtr --json -c1 192.168.1.4`
````
{
  "report": {
    "mtr": {
      "src": "qtip.private",
      "dst": "192.168.1.4",
      "tos": "0x0",
      "psize": "64",
      "bitpattern": "0x00",
      "tests": "1"
    },
    "hubs": [  }
}
````
mtr 0.87+git:a4ff54cd

**Comment by matt-kimball** [comment]
Indeed.  It repros with the latest commit, too.  The probems seems to be that `json_close` in `ui/report.c` doesn't handle `net_min(ctl) == net_max(ctl)`.

**Comment by mananas** [comment]
Seems this fix is not in version 0.92 (Ubuntu 18.04). Same problem for JSON persist.

**Comment by krisl** [comment]
and remains fixed after #355, can this issue be closed now?

**Comment by jakkanen** [comment]
Verified fixed with mtr 0.94
